src/Bicep.Core.UnitTests/BicepTestConstants.cs

```diff
@@ -133,5 +133,29 @@ public static Mock<ITelemetryProvider> CreateMockTelemetryProvider()
 
             return telemetryProvider;
         }
+
+        public static BinaryData BicepProviderManifestWithEmptyTypesLayer = BinaryData.FromString($$"""
+{
```

Comment:
[nit]: With triple-quoted strings, the placement of the closing `"""` determines the lefthand margin of the quoted string. No need to break the indentation flow and push it all the way back to the file's lefthand margin.